src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(7 * $day-size + 6 * $gap + 2 * $padding);
```

Comment:
The variables `$day-size`, `$gap`, and `$padding` are used here, but it's not clear if they are defined elsewhere in your SCSS files. Make sure these variables are defined before they are used, otherwise, the SCSS will not compile correctly.